analyzers/tests/SonarAnalyzer.Test/Rules/Utilities/SymbolReferenceAnalyzerTest.cs

```diff
@@ -352,6 +357,10 @@ private VerifierBuilder CreateBuilder(ProjectType projectType, params string[] f
                       .Should().BeEquivalentTo(assertedDeclarationLineReferences);
         }
 
+        private static void VerifyReferencesColumns(IReadOnlyList<SymbolReferenceInfo.Types.SymbolReference> symbolReference, int declarationLine, int startLine, int endLine, int startOffset, int endOffset) =>
```

Comment:
This violates S103 and should be
```suggestion
        private static void VerifyReferencesColumns(IReadOnlyList<SymbolReferenceInfo.Types.SymbolReference> symbolReference,
                                                    int declarationLine,
                                                    int startLine,
                                                    int endLine,
                                                    int startOffset,
                                                    int endOffset) =>
```